cpp/src/slice2py/PythonUtil.cpp

```diff
@@ -2844,26 +2812,26 @@ Slice::Python::validatePythonMetadata(const UnitPtr& unit)
     // "python:identifier"
     MetadataInfo identifierInfo = {
         .validOn =
-            {typeid(Module),
-             typeid(InterfaceDecl),
-             typeid(Operation),
-             typeid(ClassDecl),
+            {typeid(Slice::Module),
```

Comment:
Validation metadata started to emit warnings, after I add `PythonCompilationKind::Module` that is the second time I got bitten by a similar error.

Would be clear to use an enum, is not like you can apply metadata to any typeid.